src/Storages/ObjectStorage/DataLakes/Iceberg/AvroSchema.h

```diff
@@ -15,27 +15,30 @@ static constexpr const char * manifest_list_v1_schema = R"(
         {"name": "partition_spec_id", "type": "int", "doc": "Spec ID used to write", "field-id": 502},
         {
             "name": "added_snapshot_id",
-            "type": ["null", "long"],
+            "type": "long",
             "doc": "Snapshot ID that added the manifest",
             "field-id": 503
         },
         {
-            "name": "added_data_files_count",
```

Comment:
I think we can't change avro schema because I took it from some official iceberg repository. If we change it we will break compatibility with specification.